dune/grid/common/ZoltanPartition.cpp

```diff
@@ -95,15 +95,20 @@ std::vector<int> zoltanGraphPartitionGridOnRoot(const CpGrid& cpgrid,
     {
         parts[exportLocalGids[i]] = exportProcs[i];
     }
-    //#ifndef NDEBUG
-    if( eclipseState )
+    if( eclipseState && ! pretendEmptyGrid )
     {
+        grid_and_wells->postProcessPartitioningForWells(parts);
+#ifndef NDEBUG
         int index = 0;
         for( auto well : grid_and_wells->getWellsGraph() )
         {
+            if( ! well.size() )
```

Comment:
Is this block necessary? I'd think that for an empty `well` the code would work as is?